envoy/envoy.yaml

```diff
@@ -0,0 +1,56 @@
+admin:
+  access_log_path: /tmp/admin_access.log
+  address:
+    socket_address: { address: 0.0.0.0, port_value: 9901 }
```

Comment:
Am I right to say that this is just for logging purposes? Is there any reason why the port is 9901?

Reply:
Hmm, it's not just logging. If you access localhost:9901, it also provides some sort of admin dashboard (though probably overkill for our usecase). 

I don't think there's any particular reason why it's 9901. But this seems to be the standard port based on the configs I've seen online

Reply:
Ah I see, thanks!